src/layout/Paragraph/ParagraphComponent.tsx

```diff
@@ -28,7 +28,7 @@ export function ParagraphComponent({ node }: IParagraphProps) {
       >
         <Lang id={textResourceBindings?.title} />
       </Paragraph>
-      {textResourceBindings?.help && (
+      {textResourceBindings?.help && elementAsString(helpText) !== '' && (
```

Comment:
Hmm, I'm under the impression that if `textResourceBindings?.help` === `""`, the help text would not render here anyway. Doesn't the test pass even without the code changes? And from looking at the original bug (#1940) the problem occurs in the `Input` layout component, not the `Paragraph` layout component. The label for most layout components renders with the `GenericComponentLabel` React component.

Anyway, from looking at #1940 I'm not entirely sure the original bug is quite what it seems.. :thinking: This was reported because the preview in Studio showed the helpText button when, apparently, the text was removed - but does that happen outside the Studio preview in normal apps as well? I believe further investigation is needed.